linkedin_scraper/person.py

```diff
@@ -115,7 +115,7 @@ def get_experiences(self):
         self.scroll_to_bottom()
         main_list = self.wait_for_element_to_load(name="pvs-list", base=main)
         for position in main_list.find_elements(By.XPATH,"li"):
-            position = position.find_element(By.CLASS_NAME,"pvs-entity")
+            position = position.find_element(By.CLASS_NAME,"pvs-entity--padded")
```

Comment:
The get_experiences() function needs more rework, as some of the data is getting repeated for certain experiences, while some data is getting omitted for some experiences.